kubernetes/falcosidekick-deployment.yaml

```
@@ -0,0 +1,78 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: falcosidekick
+automountServiceAccountToken: true
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: falcosidekick
+  labels:
+    name: falcosidekick
+spec: 
+  ports:
+  - port: 2801
+    protocol: TCP
+    targetPort: 2801
+  selector:
+    name: falcosidekick
+  type: ClusterIP
+---
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: falcosidekick
+  labels:
+    name: falcosidekick
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: falcosidekick
+  strategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        name: falcosidekick
+    spec:
+      serviceAccount: falcosidekick
+      containers:
+        - name: falcosidekick
+          image: issif/falcosidekick:1.1.0
```

```suggestion
          image: falco/falcosidekick:2.7.2
```

falcosidekick is now an official project, we moved it's repository